get_logs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+scp -o 'StrictHostKeyChecking no' -i ./base-image/id_rsa -P 32001 gopher@localhost:.bash_profile ./
```

Comment:
This was just example how to copy files from pod. You don't have to so this step for getting logs

Reply:
done